### ISSUE TITLE
Big version jump from 0.2.0 to 0.3.2?

### ISSUE THREAD

**Issue by johanneskastl** [vash/dup#1]
Hi,

just curious, is there any special reason why the version jumped from 0.2.0 to 0.3.2? The new release was done by github actions, no longer by "vash" directly. But there are no release/tags 0.3.0 and 0.3.1 in between.

Kind Regards,
Johannes

**Comment by vash** [comment]
Hi Johannes, 
Thanks for your interest!
Krew employs an automatic version updating mechanism to their manually approved plugins.

Since I've mistakenly changed the description in a way that was too drastic for it to be automatically approved,
It was sent to human approval.
For us to not go through this approval process and have automatic updates through Krew,
I've re-tagged the build twice, which leads us to this version.
In order to not confuse anyone regarding the versioning, I've removed the two previous builds since they're providing the exact same features and code as this one.

Also, regarding the deployment through GitHub Actions - this is a requirement in order to get the plugin automatically updated, so from now onward, It'll be smooth sailing.

Hope I've managed to answer your question :)
